pom.xml

```diff
@@ -25,6 +25,18 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.11.2</version>
+        </dependency>
```

Comment:
why do you need this dependency?

Reply:
It's redundant. I was testing smth and forgot to take that off